bundles/framework/divmanazer/instance.js

```diff
@@ -139,6 +139,16 @@ Oskari.clazz.define('Oskari.userinterface.bundle.ui.UserInterfaceBundleInstance'
                 me.requestHandlers.modal
             );
 
+            // Hide navigation if starts in mobile mode
+            const nav = [...Oskari.dom.getRootEl().children].find(c => c.localName === 'nav');
+            if (nav) {
+                if (Oskari.util.isMobile()) {
+                    nav.style.display = 'none';
```

Comment:
We can start the plugin here in the same way as for example in here:
https://github.com/oskariorg/oskari-frontend/blob/master/bundles/framework/featuredata2/instance.js#L104-L106

This way we don't need to make changes to the mapfull bundle nor AbstractMapModule.